### ISSUE TITLE
Can not hide app list with standard user account

### ISSUE THREAD

**Issue by ReikoOnCloud** [valinet/ExplorerPatcher#3635]
Hello developer,
I I modify this option to hide the app list in start menu, a UAC window popped up, but nothing happen after checking Yes. After investigation, I found that only standard user account trigger this issue, while EP runs normally on administrative user account, and all changes made by standard users account to this option will affect administrator users. 
![图片](https://github.com/user-attachments/assets/267c689d-045c-43b7-b76c-4e07716ecb9e)
Expected behavior:  every user can edit their own EP options
Equipment information:
- Win11 23H2 22631.4037
- EP Version 22621.3527.65.5

**Comment by pyrates999** [comment]
Please try installing the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases

**Comment by Amrsatrio** [comment]
@pyrates999 This problem is not mentioned in the changelogs so suggesting to install the prerelease is useless.

**Comment by Panzimy** [comment]
> Hello developer, I I modify this option to hide the app list in start menu, a UAC window popped up, but nothing happen after checking Yes. After investigation, I found that only standard user account trigger this issue, while EP runs normally on administrative user account, and all changes made by standard users account to this option will affect administrator users. 

Yeah I have founded that some of functions need administrator. And I found some problems with apps what can not run somrthing without Administrator and if you enter a admin username and password and you will touch the Yes button the program will not run I will say that this isn't fixable

**Comment by pyrates999** [comment]
You can make the administrative user account prompt for UAC just as much as the regular user account, some settings just have it set where the UAC prompt is not set to be shown and to be automatically accepted.

Please list the settings in EP that will prompt for UAC under a normal user account when they do not show a UAC prompt under an administrative user account.
